### ISSUE TITLE
Azure AI Projects: Agent streaming does not provide information about run completed state

### ISSUE THREAD

**Issue by jhakulin** [Azure/azure-sdk-for-python#38918]
- **Package Name**: azure-ai-projects
- **Package Version**: 1.0.0b3
- **Operating System**: Windows
- **Python Version**: 3.12.8

**Describe the bug**
When I run e.g. https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ai/azure-ai-projects/samples/agents/async_samples/sample_agents_stream_eventhandler_async.py and use following code on thread run

```
    async def on_thread_run(self, run: "ThreadRun") -> None:
        print(f"ThreadRun status: {run.status}")

        if run.status == "completed":
            print(f"Run completed. Usage, completion: {run.usage.completion_tokens}, prompt_tokens: {run.usage.prompt_tokens}, total_tokens: {run.usage.total_tokens}")
```
 I cannot get completed run information e.g. for usage at the end of the stream. Looks like the data is missing.

**To Reproduce**
1. Run the sample mentioned above with changes listed

**Expected behavior**
I get run completed status information

**Screenshots**

**Additional context**

**Comment by jhakulin** [comment]
This will be fixed in 1.0.0b4 version soon

**Comment by jhakulin** [comment]
released in https://pypi.org/project/azure-ai-projects/1.0.0b4/
